text/1211-mir.md

```diff
@@ -230,19 +229,19 @@ themselves, or else they may be additional `H::Expr` nodes. This
 allows the tree that is exported to differ in small ways from the
 actual tree within the compiler; the primary intention is to use this
 to model "adjustments" like autoderef. The code to convert from our
-current AST to the HIR is not yet complete, but it can be found in the
-[`tcx` module][tcx].
+current AST to the HIR is not yet complete, but it can be found
+[here][hair-mod].
```

Comment:
My understanding is that the RFC is talking about HAIR and not HIR. I guess the text should reflect that! :) I'll make another change.